packages/nextjs/components/scaffold-eth/Contract/utilsDisplay.tsx

```diff
@@ -22,8 +22,19 @@ export const displayTxResult = (
   }
 
   // TODO: Handle too big bigNumber to show ethers value -> https://github.com/scaffold-eth/scaffold-eth-2/commit/e5a591f820851f919632f3343d1f3f4548f07ac7#diff-5ca89ee1d9a40426f20c02e8b49a214feace564a3dc64907d5d4c5de90dcde03
-  if (typeof displayContent === "bigint" || typeof displayContent === "number") {
-    return displayContent.toString();
+  if (typeof displayContent === "bigint") {
+    try {
+      console.log("displayContent", displayContent);
+      console.log("Number.MAX_SAFE_INTEGER", displayContent.toString());
+      const asNumber = Number(displayContent);
+      if (asNumber <= Number.MAX_SAFE_INTEGER && asNumber >= Number.MIN_SAFE_INTEGER) {
+        return asNumber;
+      } else {
+        return "Ξ" + formatEther(displayContent);
+      }
+    } catch (e) {
+      return "Ξ" + formatEther(displayContent);
+    }
```

Comment:
I don't understand this change. A `bigint` display value can represent anything, not only an ether value. So I don't think we should do anything besides turning it into a string here.

Reply:
Actually, this was in `main` and also in SE-1 so I added it : 
https://github.com/scaffold-eth/scaffold-eth-2/blob/123e6d84ac613e15e07fd854268b6869d7353615/packages/nextjs/components/scaffold-eth/Contract/utilsDisplay.tsx#L21

I think its there to nicely format stuff like `balanceOf` although `Ξ` is kind of misleading and every token might don't have `18` decimal places but it does its jobs pretty well.....since if the number is too large it's most likely representing some kind of decimal or balance 

But spawning @carletex here since I am also not too sure whether we should remove or keep it